src/sql/media/icons/common-icons.css

```diff
@@ -652,3 +652,23 @@ Includes non-masked style declarations. */
 	width: 32px;
 	height: 32px;
 }
+
+/** Info icons **/
+
+.vs .optionsDialog-options .optionsDialog-description .modal-title {
```

Comment:
You still have feature-specific classes in here. This should only contain generic icons for the entire app to use - scoping them down to specific areas defeats the purpose of having this in the first place. 

So the only two selectors should be 

```
.vs-dark .info-icon.codicon
.hc-black .info-icon.codicon
```

and 

`.info-icon.codicon`